.github/workflows/ci.yml

```diff
@@ -81,8 +81,47 @@ jobs:
             - name: Run codestyle checks
               run: composer cs-check
 
-            - name: Run codeception tests (failing because of missing `spryker/oauth` dependency)
-              run: composer test || true
+            - name: Run codeception tests
+              run: composer test
 
-            - name: Run PHPStan (failing because of missing `spryker/oauth` dependency)
+            - name: Run PHPStan
+              run: composer stan
+
+    lowest:
+        name: Prefer Lowest
+        runs-on: ubuntu-18.04
+
+        steps:
+            - name: Checkout@v2
+              uses: actions/checkout@v3
+
+            - name: Validate composer.json and composer.lock
+              run: composer validate
+
+            - name: Composer get cache directory
+              id: composer-cache
+              run: |
+                  echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Composer cache
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-composer-
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '8.0'
+                  extensions: mbstring, intl, bcmath
+
+            - name: Composer prefer lowest
+              run: composer lowest-setup
+
+            - name: Run PHPStan (currently not running properly)
               run: composer stan || true
```

Comment:
Can we avoid `|| true`?

Reply:
It's for all eco modules, there is an error that does not depend on us. I do not remember the exception but can remove it and show the result if it's needed

Reply:
Those hacks make CI green without it being green, I would also suggest those are only valid temporary to see if follow up tasks are OK
those should not get merged to master.

Reply:
> PHP Fatal error:  Uncaught TypeError: _HumbugBox5f943a942674\Nette\Utils\Reflection::getUseStatements(): Return value must be of type array, null returned in phar:///home/runner/work/authorization-picking-app-backend-api/authorization-picking-app-backend-api/vendor/phpstan/phpstan/phpstan/vendor/nette/utils/src/Utils/Reflection.php:180

Maybe PHPStan needs to be removed from prefer lowest if it isnt running

Reply:
Do you know which part in ci.yml is responsible for generating Transfers ets ?

Reply:
tests do that automatically via helpers afaik

thats why some modules have tests || true for phpstan to run afterwards

Reply:
This is documented in the standalone testing doc for
\SprykerTest\Shared\Transfer\Helper\TransferGenerateHelper

Reply:
That said: The whole testing harness for Spryker modules is a beast and could definitly need some serious refactor/rework

Reply:
With `tests || true` and some dependency adjustments works, thanks!
@dereuromark and @asmarovydlo please check again